cmd/maya-apiserver/app/server/backup_endpoint_v1alpha1.go

```diff
@@ -107,6 +107,10 @@ func (bOps *backupAPIOps) create() (interface{}, error) {
 		return nil, CodedError(400, fmt.Sprintf("Failed to find healthy replica"))
 	}
 
+	if bkp.Spec.LocalSnap {
```

Comment:
should we check for two healthy CVRs for localsnap?

Reply:
not required. We are creating snapshot only. So if cvr is not healthy then snapshot creation will fail.

Reply:
correct.. not required